test/platform-stats.test.js

```diff
@@ -280,12 +280,12 @@ describe('platform-stats', () => {
   describe('daily_participants', () => {
     it('submits daily_participants data for today', async () => {
       /** @type {Measurement[]} */
-      const honestMeasurements = [
+      const allMeasurements = [
         { ...VALID_MEASUREMENT, participantAddress: '0x10' },
         { ...VALID_MEASUREMENT, participantAddress: '0x10' },
         { ...VALID_MEASUREMENT, participantAddress: '0x20' }
       ]
-      await updatePlatformStats(pgClient, honestMeasurements)
+      await updatePlatformStats(pgClient, allMeasurements)
```

Comment:
updatePlatformStats is defined here:

https://github.com/filecoin-station/spark-evaluate/blob/670984007ce2b64177281a1f98ebe611274c8132/lib/platform-stats.js#L8-L12